src/Data/Fin/Properties.agda

```diff
@@ -100,6 +103,18 @@ zero  ≟ suc y = no λ()
 suc x ≟ zero  = no λ()
 suc x ≟ suc y = map′ (cong suc) suc-injective (x ≟ y)
 
+≡-irrelevant : Irrelevant {A = Fin n} _≡_
+≡-irrelevant = Decidable⇒UIP.≡-irrelevant _≟_
+
+≟-diag : (eq : i ≡ j) → (i ≟ j) ≡ yes eq
+≟-diag = ≡-≟-identity _≟_
```

Comment:
I would vote for calling this `≟-diag-≡` and `≟-off-diag` as `≟-diag-≢` for consistency? Actually do we even need the `diag` in the names? Doesn't really do much...

Reply:
That works for me